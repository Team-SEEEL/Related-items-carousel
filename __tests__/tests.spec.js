const axios = require('axios');

describe('Api test', () => {
  test('Ajax request to endpoint /api/products should return data', () => {
    return axios.get('http://localhost:3000/api/products')
      .then((results) => {
        expect(results.data.length).toBe(100);
      });
  });
});
