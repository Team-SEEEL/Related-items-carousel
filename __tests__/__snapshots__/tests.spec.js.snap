// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A suite Should render correctly 1`] = `ShallowWrapper {}`;
